admin_guide/diagnostics_tool.adoc

```diff
@@ -218,14 +227,15 @@ using the provided *_health.yml_* playbook.
 
 [WARNING]
 ====
-Due to potential changes the health check playbooks could make to hosts, they
-should only be used on clusters that have been deployed using Ansible and using
-the same inventory file with which it was deployed. Changes mostly involve
-installing dependencies so that the checks can gather required information, but
-it is possible for certain system components (for example, `docker` or
-networking) to be altered if their current state differs from the configuration
-in the inventory file. Only run these health checks if you would not expect your
-inventory file to make any changes to your current cluster configuration.
+Due to potential changes the health check playbooks could make to the
+environment, the playbooks should only be run against clusters that have been
```

Comment:
s/the playbooks should only be run against clusters that have been deployed using Ansible with the same inventory file used during deployment/you must run the playbooks against only clusters that you deployed using Ansible and use the same inventory file that you used during deployment.